.travis.yml

```diff
@@ -11,12 +12,15 @@ osx: &osx
 matrix:
    include:
       - <<: *linux
+        env: CONAN_CLANG_VERSIONS=4.0 CONAN_DOCKER_IMAGE=lasote/conanclang40
```

Comment:
It should use the newer `conanio/clang40` container

Reply:
that's true! thanks!